packages/conjure-client/.npmignore

```diff
@@ -0,0 +1,7 @@
+**/*
+!bin/*
+!lib/**/*
+!src/**/*
+**/__tests__/*
+**/__mocks__/*
+**/__integTest__/*
```

Comment:
Shall we also delete the root level npmignore as it doesn't really do anything anymore?

Reply:
sure i can do that

Reply:
(lol github now tracks this as a move, in case you are confused like i was for a hot second)